lsp-ui-doc.el

```diff
@@ -763,6 +756,8 @@ before, or if the new window is the minibuffer."
   "Minor mode for showing hover information in child frame."
   :init-value nil
   :group lsp-ui-doc
+  (unless (posframe-workable-p)
```

Comment:
Aha - lsp-ui doc display works in my terminal as I have `lsp-ui-doc-use-childframe` set to `nil`, and that tells lsp-ui to use an inline rendering method. 

You need to allow for that possibility in this check.

Reply:
Ah, good call - that check would block that one. I'll fix that up.

Reply:
I think ivy-posframe does it's check each time it's invoked, so if you run emacs daemon with both GUI and TTY clients, it uses posframe in the GUI and falls back to whatever its other method is when invoked from TTY. Maybe you could do that here also, rather than a single up-front check?

Reply:
Just pushed a change that should do that.